classes/AlertView/Standard.php

```diff
@@ -409,6 +410,10 @@ private function searchForConstituenciesAndMembers() {
             $members_from_words = \MySociety\TheyWorkForYou\Utility\Search::searchMemberDbLookupWithNames($text, true);
             $this->data['members'] = array_merge($members_from_words, $members_from_names);
             [$this->data['constituencies'], $this->data['valid_postcode']] = \MySociety\TheyWorkForYou\Utility\Search::searchConstituenciesByQuery($text, false);
+            if (!$this->data['valid_postcode']) {
+                $this->data['member_constituencies'] = \MySociety\TheyWorkForYou\Utility\Search::searchMembersByConstituency($text);
```

Comment:
I realise /search looks like it only searches MPs by constituency name at the moment, but (not for now, but please ticket), really we should only have the one way to search for reps by constituency name, not this new function and an entirely different way on search results page.